### PR TITLE
Format dates with current locale

### PR DESCRIPTION
**Pull request by pablopunk**
SDCP-207

moment.js already provides abbreviations for months

### CODE REVIEW

File: assets/utils.js [hunk 1]
```diff
@@ -17,6 +18,10 @@ import {
     TO_BE_CONFIRMED_TEXT
 } from './agenda/utils';
 
+// Add locales to moment.js
+moment.locale('fr_CA');
+moment.locale('en');
```
Comment (petrjasek) [inline assets/utils.js:23]: shouldn't it be based on the config rather than hardcoding it here?

Comment (pablopunk) [reply assets/utils.js:23]: I tried to load the config from `window` object here but it's not available yet

Comment (petrjasek) [reply assets/utils.js:23]: ok

Comment (pablopunk) [reply assets/utils.js:23]: also if we add another locale, we would still need to add it manually [here](https://github.com/superdesk/newsroom/pull/1036/files#diff-c6a35b21c13bfe61f8ac39d33b8d771dR12) cause that's how moment works 🤷‍♂

Comment (petrjasek) [reply assets/utils.js:23]: would be good to leave a comment regarding this imo

**Pull request merged**